home.html

```diff
@@ -48,35 +49,48 @@
       <header>
         <div class="tools">
             <div>
-                <a>MSK Markdown</a>
-                <button onclick="apply('bold')" title="Bold">B</button>
-                <button onclick="apply('italic')" title="Italic"><i>I</i></button>
-                <button onclick="apply('strike')" title="Strikethrough"><strike>S</strike></button>
+                <a>MSK Markdown</a>&nbsp;&nbsp;
+                <div class="dropdown">
+                    <button class="drop-header">Header</button>
+                    <div class="dropdown-content">
+                        <button class="drop-button" onclick="apply('h1')" title="Header 1">H1</button><br>
+                        <button class="drop-button" onclick="apply('h2')" title="Header 2">H2</button><br>
+                        <button class="drop-button" onclick="apply('h3')" title="Header 3">H3</button><br>
+                        <button class="drop-button" onclick="apply('h4')" title="Header 4">H4</button><br>
+                        <button class="drop-button" onclick="apply('h5')" title="Header 5">H5</button><br>
+                        <button class="drop-button" onclick="apply('h6')" title="Header 6">H6</button>
```

Comment:
Not directly related to the header change, but it's a good practice to separate JS from HTML - so you need to remove `onclick` and use `.addEventListener` on a separate JS file.

Reply:
So, I will have to use `.addEventListener` instead of  `onclick`. Thank you. :)

Reply:
yeah, don't ever use inline attributes for setting up listeners - it is a whole can of worms on its own. That really saddens me that web frameworks these days help developers internalize the wrong thing to do (frameworks like React use those for different reasons because under the hood those are not inline attributes, but still)